views/scan.hbs

```diff
@@ -0,0 +1,44 @@
+{{!< default }}
+
+    <div class="top-bar">
+      <div class="top-bar-left">
+        <h1><a href="/">Firefox Breach Alerts</a></h1>
+      </div>
+      <div class="top-bar-right">
+        <ul class="menu">
+          <li><a href="/">About</a></li>
+          <li><a href="#tips">Tips</a></li>
+          <li><a class="button" data-open="subscribe-modal">Subscribe</a></li>
+        </ul>
+      </div>
+    </div>
```

Comment:
Similarly, this block seems the same as https://github.com/mozilla/blurts-server/pull/73/files#diff-47ab88469c3f63d8b827d9130935ae0fR2

Not sure if we should move into "default.hbs" template, or into a reusable partial.